react-ui-ts/src/Components/ExercisePage/Solution.tsx

```diff
@@ -0,0 +1,4 @@
+export const Solution = () =>
+{
```

Comment:
Higher order functions usually have a starting bracket on the same line of the "=>".